public/app/core/angular_wrappers.ts

```diff
@@ -17,7 +17,7 @@ import {
 } from '@grafana/ui';
 import { FunctionEditor } from 'app/plugins/datasource/graphite/FunctionEditor';
 import { SearchField } from './components/search/SearchField';
-import { GraphContextMenu } from 'app/plugins/panel/graph/GraphContextMenu';
+import { GraphContextMenu } from '@grafana/ui';
```

Comment:
Can we move this import to the imports from grafana/ui above?

Reply:
yes, good catch!